astropy/time/tests/test_comparisons.py

```diff
@@ -30,8 +30,8 @@ def test_miscompares(self):
                 op(t1, None)
             assert str(err).endswith("Unsupported operand type(s) for {0}: 'Time' and 'NoneType'"
                                      .format(op_str))
-        assert (t1 == None) is False
-        assert (t1 != None) is True
+        assert (t1 is None) is False
+        assert (t1 is not None) is True
```

Comment:
That would be Tom A or me. Please keep these as `==` and `!=` --> they are specifically meant to test `Time.__eq__` and `Time.__ne__` -- and, no, we cannot count on everybody to stick to PEP8!
If you want to avoid disabling pylint, it would be fine to rewrite as
```
assert t1.__eq__(None) is False
```
etc. (though I think it is slightly less clear, and I dislike a bit rewriting tests just so PEP8 passes).